docs/guides/manage-identities/account-recovery.mdx

```diff
@@ -0,0 +1,187 @@
+---
+id: account-recovery
+title: Administrative Account Recovery
+---
+
+It is possible to initiate account recovery for identities using the Admin API
+endpoints.
+
+:::note
+
+It is possible to generate a link for an account without a recovery address via
+the admin API, but if the recovery link expires the user won't be able to
+re-initiate the flow by himself as long as the recovery address has been added.
+
+:::
+
+To create the account recovery link, use:
+
+````mdx-code-block
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+import CodeBlock from '@theme/CodeBlock'
+import SelfHostedKratos from '@site/src/components/SelfHostedKratos'
+
+<Tabs
+  defaultValue="curl"
+  values={[
+    {label: 'curl', value: 'curl'},
+    {label: 'GoLang', value: 'go'},
+    {label: 'Self-Hosted Ory Kratos', value: 'kratos'},
+  ]}>
+  <TabItem value="curl">
+
+```shell
+curl --request POST -sL \
+  --header "Authorization: Bearer ory_pat_xRKLsFEOUFQFVBjd6o3FQDifaLYhabGd" \
+  --header "Content-Type: application/json" \
+  --request POST \
+  --data '{
+  "expires_in": "12h",
+  "identity_id": "e01b5f2f-6afc-4194-8578-4cebcf69a4d5"
+}' https://playground.projects.oryapis.com/api/kratos/admin/recovery/link
+```
+
+  </TabItem>
+  <TabItem value="go">
+
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/ory/client-go/client"
+	"github.com/ory/client-go/client/admin"
+	"github.com/ory/client-go/models"
+)
+
+func main() {
+	c := client.New(nil, &client.TransportConfig{
+        Host: "https://playground.projects.oryapis.com",
+        BasePath: "/",
+        Schemes: []string{"http"},
+    })
+
+	res, err := c.Admin.CreateRecoveryLink(admin.NewCreateRecoveryLinkParams().WithBody(admin.CreateRecoveryLinkBody{
+		IdentityID: models.UUID("the-uuid"),
+	}))
+	if err != nil {
+		// ...
+	}
+
+	fmt.Printf("Use link: %s", *res.Payload.RecoveryLink)
+}
+```
+
+  </TabItem>
+  <TabItem value="kratos">
+    <SelfHostedKratos />
+  </TabItem>
+</Tabs>
+````
+
+The response contains a `recovery_link` value which is the link the user should
+use to set up his or her credentials (e.g. connect to a Social Sign In Provider,
+set up a password, ...):
```

Comment:
```suggestion
The response contains a `recovery_link` value. This is the link the user should
use to set up his or her credentials to connect to a Social Sign In Provider or
set up a password :
```

Again as above we have to decide how we spell "Social Sign In Provider" to make it consistent.

Also latin abbrevations and parantheses.